### PR TITLE
added comment to FormSelect component

### PR DESCRIPTION
**Pull request by onlinehub88**

### CODE REVIEW

**Review by seniordev0207 — COMMENTED** [review]
Done

File: components/FormSelect/index.tsx
Comment (onlinehub0808) [inline components/FormSelect/index.tsx:1]: done

**Review by onlinehub0808 — COMMENTED** [review]
reviewed

**Review by onlinehub88 — COMMENTED** [review]
started

**Review by FamousTiger — COMMENTED** [review]
done

**Pull request merged**